cvm/quantization/sym_pass.py

```diff
@@ -735,24 +735,79 @@ def _reduce_graph(sym, params, graph, inputs_ext):
     return node, params
 
 def fuse_transpose(symbol, params, logger=logging):
-    # node                           node            node
-    #  |                 reduce       |
-    # transpose1       =========>  transpose    or
-    #  |
-    # transpose2
     def _fuse_transpose(sym, params, graph, inputs_ext):
-        if sym.attr('op_name') == 'transpose':
-            axes, consec = eval(sym.attr('axes')), False
-            childs = sym_iter(sym.get_children())
+        name, op = sym.attr('name'), sym.attr('op_name')
+        childs = sym_iter(sym.get_children())
+
+        #   node                                 node              node
+        #     |                reduce              |
+        # transpose1         =========>        transpose3     or
+        #     |
+        # transpose2
+        if op == 'transpose':
+            axes = eval(sym.attr('axes'))
             if childs[0].attr('op_name') == 'transpose':
                 caxes = eval(childs[0].attr('axes'))
                 axes = tuple([caxes[ii] for ii in axes])
-                consec = True
-            if consec:
                 sym = sym_iter(childs[0].get_children())[0]
                 if axes != tuple(sorted(axes)):
                     name = sym.attr('name') + "_fusetranspose"
                     sym = mx.sym.transpose(sym, axes=axes, name=name)
+
+        #   node                                 node
+        #     |                switch              |
+        # transpose1         =========>          relu
+        #     |                                    |
+        #   relu                               transpose1
+        elif op == 'relu':
+            if childs[0].attr('op_name') == 'transpose':
+                axes, name = eval(childs[0].attr('axes')), childs[0].attr('name')
+                sym = sym_iter(childs[0].get_children())[0]
+                sym = mx.sym.relu(sym)
+                sym = mx.sym.transpose(sym, axes=axes, name=name)
+
+        #     node     ...    node
+        #       |               |
+        #   transpose1 ...  transpose1                     node    node
+        #       \               /              reduce        \       /
+        #             concat                 =========>        concat
+        #                                                        |
+        #                                                    transpose1
+        elif op == 'Concat':
+            same, axeses = True, set()
+            for child in childs:
+                if child.attr('op_name') != 'transpose':
+                    same = False
+                    break
+                axeses.add(child.attr('axes'))
+            if same and len(axeses) == 1:
+                clist, dim = [], int(sym.attr('dim'))
+                axes = eval(list(axeses)[0])
```

Comment:
no need to eval since `get_attr` has evaluated.

Reply:
already fixed